docs/contributing/index.md

```diff
@@ -53,35 +79,167 @@ Tap, click, or hover over the GitHub Octocat icon for options.
+    You are welcome to fix as many errors as you like.
+    You are only responsible to fix errors that you create.
+    ```
+
+    ```{note}
+    Eventually the `spellcheck` build will be required, but not at this time.
+    We welcome improvements to the dictionary.
+    ```
+
+    ```{seealso}
+    {ref}`setup-build-available-documentation-builds-label`.
+    ```
+
+1.  After the builds pass, commit changes to your branch, and push it to the remote repository on GitHub.
+    The remote repository may be either a branch in your fork of the project or a branch in the `plone/documentation` upstream repository.
+
+    ```shell
+    git commit -m "My descriptive commit message"
+    git push
+    ```
+
+1.  Visit the GitHub `documentation` repository, and [create a pull request](https://github.com/plone/documentation/compare) against the branch `6-dev`.
+1.  Members who subscribe to the repository will receive a notification and review your request.
+
+
+(contributing-documentation-and-volto-label)=
+
+### Working with both the `documentation` and `volto` repositories
+
+When you want to edit files within the `docs/volto` subdirectory, the process is slightly different.
+We use git submodules to manage multiple repositories.
+We imported the `volto` repository into the `documentation` repository as described in {doc}`setup-build`.
+
+```{important}
+We currently use the branches `plone/documentation@6-dev` and `plone/volto@plone6-docs` as the main branches for developing Plnoe 6 Documentation.
+These branches may change as we get closer to a production release.
+```
+
+1.  From the project root directory, synch your local `6-dev` branch with its remote.
+    You might need to resolve conflicts.
+
+    ```shell
+    git checkout 6-dev
+    git pull
+    ```
+
+1.  Change your working directory from the project root directory to `submodules/volto`.
+
+    ```shell
+    cd submodules/volto
+    ```
+
+1.  Update the submodule, and synch your local `plone6-docs` branch with its remote.
+    You might need to resolve conflicts.
+
+    ```shell
+    git submodule update
+    git checkout plone6-docs
+    git pull
+    ```
+
+1.  Create a new branch from `volto/plone6-docs`.
+
+    ```shell
+    git checkout -b <new_branch>
+    ```
+
+1.  Make edits to files in `docs/volto` using your favorite editor, and save, preview, and test.
+    You must run and pass the builds `html` and `linkcheck` without causing new errors.
+
+    ```shell
+    # Optionally clean the builds to avoid cache issues
+    make clean
+    make html
+    make linkcheck
+    ```
+
+1.  Back in `submodules/volto`, commit and push your changes to the remote.
+
+    ```shell
+    git add <files>
+    git commit -m "My commit message"
+    git push
+    ```
+
+1.  Now return to the project root directory, and update the submodule to point to the commit you just made, and push your changes to the remote repository.
```

Comment:
```suggestion
1.  Now return to the project root directory, and update the submodule to point to the commit you just made, and push your changes to the remote /plone/documentation repository.
```

Reply:
The remote branch could be in either a user's fork or the upstream repo `/plone/documentation`. Not being explicit allows either usage.